rascal-textmate-core/src/main/rascal/VSCodeRascal.rsc

```diff
@@ -0,0 +1,61 @@
+@synopsis{
+    Main function to generate a new TextMate grammar for Rascal, to be used in
+    the special VS Code extension
+}
+
+module VSCodeRascal
```

Comment:
This is essentially a rename of the old `VSCode` module:

  - Function `main` was a bit streamlined
  - Function `getRscGrammar` is identical to before